---
output: 
    thesisdown::thesis_pdf: default
bibliography: bib/library.bib
csl: csl/nature.csl
space_betwee_paragraphs: true
fig_caption: true
always_allow_html: yes
link-citations: true
toc-depth: 2
lot: true
lof: true
indent: true
header-includes: # include other LaTeX packages here
    \usepackage{booktabs}
    \usepackage{longtable}
    \usepackage{siunitx}
    \usepackage[left]{lineno}
    \usepackage{float}
    \floatplacement{figure}{H}
    \linenumbers
    \pagestyle{plain}
    \raggedbottom 
    \usepackage{indentfirst}
editor_options: 
  chunk_output_type: console
---

```{r, echo=FALSE, message=FALSE}
source("~/OneDrive - University of Bristol/PhD/Main project/Thesis/index/packages.R")
```
# Further elucidating the effect of adiposity on the plasma proteome using a weight loss intervention {#BMI-protein-RCT}
This work has been presented at the International Genetic Epidemiology Society meeting 2021 and the abstract is published in the journal Genetic Epidemiology[@IGES2021]. I am the first author of this abstract and was the presenting author of the work.

## Background

Obesity is associated with an increased risk of type II diabetes (T2D), CVD, musculoskeletal diseases, and types of cancer[@Garg2014; @Khan2018; @Kortt2002]. These associations are well established, however mechanisms of disease are less clear. It is likely that a change in the composition of proteins circulating in the blood plays a role in obesity-related cardiovascular risk[@Goudswaard2021]. The previous results chapter (Chapter \@ref(BMI-protein-MR)) examined the effect of BMI on the plasma proteome and provided estimates for the average change in proteins per average difference in BMI in a general population, using observational estimates and one sample Mendelian randomisation (MR). Results suggested a BMI effect on proteins such as leptin, sex hormone binding globulin (SHBG), fatty acid binding protein 4 (FABP4) and fumarylacetoacetase. Within an enrichment analysis, this change in circulating levels of these proteins was implicated in CVDs. There is also evidence that obesity affects platelet function, thereby elevating the risk of thrombotic events. Previous work and results from Chapter \@ref(chemokine-platelets) suggest that circulating proteins could mediate this risk.

Although MR helps to overcome issues inherent to observational studies such as confounding and reverse causation, MR has its own limitations. For example, it is possible that genetic variants used for the genetic risk score could affect protein levels through routes other than changes in BMI. This is known as horizonal pleiotropy[@Davies2018]. As well as this, although there were over 2700 particpants within the MR, this is a relatively small sample size for the study design. Additionally, it is important to not only look at which proteins are altered within the population with higher BMI; identifying which proteins can be modulated with weight loss is important as this may help reveal mechanistic insight into mechanisms of obesity-related CVD. Triangulation can therefore be used in efforts to further determine the BMI effects on the plasma proteome. Triangulation is defined as strengthening causal inference through the combination of study designs which each have separate sources of bias[@Lawlor2016]. Therefore, to add an independent source of evidence to the MR in the previous chapter, the Diabetes REmission Clinical Trial (DiRECT) can be utilised. The DiRECT trial consisted of a group of patients with overweight/obesity and T2D who were either given guideline T2D care or an acute intervention, consisting of a low calorie diet, to help with weight loss and T2D remission[@Lean2018]. This chapter therefore aims to: examine the effect of reducing BMI with caloric restriction on the plasma proteome.

(ref:caption-BMI-protein-graphic2) **Schematic of the associations explored in Chapter \@ref(BMI-protein-RCT). This chapter explores the effect of caloric restriction-induced weight loss on the plasma proteome.** Figure made using BioRender.com.
(ref:scaption-BMI-protein-graphic2) Schematic of the associations explored in Chapter \@ref(BMI-protein-RCT)
```{r BMI-protein-graphic2, echo = FALSE , fig.align = "center", fig.scap="(ref:scaption-BMI-protein-graphic2)", fig.cap="(ref:caption-BMI-protein-graphic2)", out.width="80%"}
include_graphics(path = "figure/DiRECT/Thesis_graphic_overview_BMI_protein_CVD.png", dpi = NA)
```

## Methods

### Study design and participants
Samples analysed within this study were collected from participants enrolled in the Diabetes Remission clinical trial (DiRECT). DiRECT was a cluster-randomised trial which took place at 49 primary care practices in Scotland and Tyneside[@Lean2018]. Ethical approval was granted from West 3 Ethics Committee (January, 2014) and the National Health Service (NHS). Participants were recruited between 25th July 2014 and 5th August 2016. Details of the protocol have previously been published[@Leslie2016]. Participants enrolled were between 20-65 years, diagnosed with T2D within previous 6 years, had a BMI of between 27-45kg/m^2^. Participants were excluded if they were: using insulin, had a HbA1c concentration of ≥12 % (≥108 mmol/mol), more than 5kg weight loss in the preceding six months, an estimated glomerular filtration rate of <30 mL/min per 1.732 m^2^. Other exclusion criteria include malignancy, heart failure, recent MI (<6 months), enrollment in other clinical trials, addiction to illegal drugs, difficulty in understanding the study, current use of drugs to treat obesity, eating disorders, pregnancy or admission to hospital for depression or use of antipsychotic medication[@Lean2018]. Participants in the control group received best-practice care by guidelines. The intervention group followed the Counterweight-Plus weight management programme[@Lean2013]. This programme a involved total diet replacement (TDR) phase using a low energy diet (825-853 kcal/day) for 3-5 months. Following the TDR, there was a structured food reintroduction phase of 2-8 weeks. Participants then attended monthly weight loss maintenance visits. Those in the intervention group had their antidiabetic and antihypertensive drugs discontinued. In total, there were 146 patients in the control group and 119 in the intervention group who remained enrolled in the study at 1 year, as detailed in Figure \@ref(fig:direct-participants).

(ref:scaption-direct) Overview of participants in the DiRECT trial
(ref:caption-direct) **Overview of participants in the DiRECT trial.** Figure made using BioRender.com.

```{r direct-participants, echo = FALSE, fig.cap="(ref:caption-direct)", fig.align = "center", fig.scap= "(ref:scaption-direct)"}
include_graphics(path = "figure/DiRECT/DiRECT_study_summary.png", dpi = NA)

```

### Variables included in analysis
Age and sex were self-reported. Height was measured with the Frankfort plane horizontal, with a portable stadiometer (Chasmors Ltd, London). Weight was measured using Class 111 approved calibrated scales (Marsden Group UK)[@Leslie2016]. Blood was donated at various timepoints including at baseline (week 0) and at 1 year (~week 52), where HDL cholesterol, triglycerides, HbA1c and plasma glucose were measured. Systolic blood pressure was measured with the patient seated, rested and with legs uncrossed for ≥5 mins. BMI was calculated by dividing the weight (kg) by the square of the height (m). BMI change was calculated by subtracting BMI at baseline from BMI at 1 year. This BMI change was rank normal transformed (function rntransform() from "moosefun" R package https://github.com/hughesevoanth/moosefun/). The centre attended and list size of the practice (>5700 or ≤5700) determined whether participants were allocated to control or intervention (minimisation variables) and therefore were recorded.
 
### Proteomics
Blood was taken from participants into 9 mL vacutainers with EDTA (an anticoagulant) at baseline at and 1-year post-randomisation. Blood samples were centrifuged to derive plasma samples and plasma was stored at -80°C. Protein detection was performed by the SomaScan assay by SomaLogic. As mentioned before, this technique uses Slow Off-rate Modified Aptamers (SOMAmers) which make direct contact with proteins and quantifies protein levels by using a DNA microarray[@Rohloff2014]. There were 5284 proteins included in the array, of which 4601 proteins passed internal quality control checks. The proteomic data was further cleaned using the "metaboprep" R package[@Hughes2021], with protein levels from both timepoints all QC'd together. This R package identified no extreme missingness (defined at 0.2 for sample and protein missingness), removed outliers (based on more than 5 SDs based on principal components (PCs) 1 and 2). The package also calculated the number of independent proteins by using pairwise correlation coefficients between proteins (869 representative proteins based on correlation coefficient of 0.5). The Shapiro-Wilk test was implemented to identify proteins which have a normal distribution (W≥0.95). Only 399 out of 4601 proteins had W statistics ≥0.95. As a result of this, all data was transformed to meet normality assumptions for analyses. This data log~2~ transformed to match protein transformation by Sun et al.[@Sun2018] then adjusted for age and sex (data had already undergone technical adjustment). Protein change was calculated for each individual by subtracting the baseline protein levels from protein levels at one-year. These protein change values were rank normal transformed.

### Statistical analysis
Analyses were performed using R version 3.6.1. A total of 265 participants were included in the current analysis. Participant baseline characteristics were summarised as the mean and SD. Baseline characteristics were calculated for both control group (N=146) and intervention (N=119) and were compared across groups using either a two-tailed unpaired t-test or a Chi^2^ test. 

The associations between BMI change and protein changes were explored using linear regression adjusting for centre, list size, age and sex (function lm() from base R "stats" package). Due to the nature of linear regression, outputs of regression provide the mean individual change in protein (in SDs) per SD increase in BMI. Therefore if more weight loss reduces the levels of protein, the beta coefficient is positive[@Figarska2020]. The association between potential confounders (age and sex) and both exposure (BMI change) and outcome (protein change) were explored using linear regression. The difference in BMI change in intervention group compared to control was performed using linear regression, adjusting for centre, list size, age and sex. Covariables (age, sex, centre and list size) were compared across treatment groups. Treatment group was therefore used as an instrument for BMI change in a two stage least squares analysis \@ref(fig:direct-tsls) to estimate the effects of BMI change on protein change using the ivreg() function from the "AER" R package (https://github.com/cran/AER/blob/master/R/ivreg.R). BMI change-protein change point estimates for 4601 proteins from the multiple linear regression analyses and two stage least squares analyses were compared using linear regression. 

The association between BMI change and change in platelet proteins were also examined by searching for the UniProt IDs of the same platelet proteins as the previous chapter (Chapter \@ref(BMI-protein-MR)). This approach was used due to the lack of distinct groupings in the SomaLogic protein list. As a sensitivity analysis, for proteins which associated with BMI change, baseline protein levels were compared across control and intervention groups. Similar to the previous chapter, the estimates for the effect of BMI change on a change in levels of the chemokines MDC and TARC were also explored. P-values were adjusted for multiple testing based on the number of representative proteins (0.05/869 = 5.8x10^-5^). As a separate sensitivity analysis, a linear mixed model was used to account for baseline and endpoint protein levels (as the change in protein variable derived does not account for this). For the linear mixed model, the lmer() function from the "lme4" package, protein levels were used as the dependent variable, with treatment group and visit entered as fixed effects (and exploration of an interaction) and subject as a random effect. Estimates from the interaction of treatment group and visit were derived for each protein. These estimates were compared with the two stage least squares estimates.

(ref:scaption-direct-tsls) Schematic of two stage least squares analysis performed using DiRECT data
(ref:caption-direct-tsls) **Schematic of two stage least squares analysis performed in the current chapter.** Figure made using BioRender.com.
```{r direct-tsls, echo = FALSE, fig.cap="(ref:caption-direct-tsls)", fig.align = "center", fig.scap= "(ref:scaption-direct-tsls)"}
include_graphics(path = "figure/DiRECT/DiRECT_analysis.png", dpi = NA)

```

### Disease class enrichment 
Similarly to the previous chapters, proteins which had a BMI effect were taken forward into an enrichment analysis using DAVID Bioinformatics 6.8 [@Huang2009]. In total, 144 proteins were compared with all proteins detected (4601). Enrichment was explored by using the genetic association database (GAD). Enrichment was assessed by fold enrichment and by Bonferroni-corrected p-values. StringDB[@Szklarczyk2021] was also used for a pathway enrichment analysis as DAVID Bioinformatics does not allow assigning of a rank order (such as by P value) to the input. Here the UniProt IDs of the proteins along with their -log~10~ p value (smallest first) for both the observational analysis and the instrumental variable analysis. Pathway analysis was explored using gene ontology (GO) terms and KEGG pathways[@Kanehisa2016].

## Results

### Participant characteristics

Participants displayed similar characteristics across control and intervention groups (Table \@ref(tab:DiRECT-participants)). Baseline BMI was similar across treatment groups (34.2 kg/m^2^ with SD of 4.3 kg/m^2^ in the control group vs 35.0 kg/m^2^ with SD of 4.6 kg/m^2^ in the intervention group, p=0.19). Other variables relating to cardiometabolic health were also similar across groups, including systolic blood pressure, HbA1c, glucose and insulin levels. The participants in the intervention group were younger and included fewer participants from Scotland but more from Tyneside.

(ref:caption-DiRECT-participants) **DiRECT participant characteristics**
(ref:scaption-DiRECT-participants) DiRECT participant characteristics
```{r DiRECT-participants,  echo=FALSE, warning=FALSE}
options(knitr.kable.NA = "")
var = c("Age", "Sex", "Male", "Female", "Body mass index (kg/m2)", "Systolic Blood Pressue (mmHg)", "HbA1c (mmol/mol", "Glucose (mmol/l)", "Insulin (uu/ml)", "Cholesterol (mmol/l)", "HDL (mmol/l)", "Triglycerides (mmol/l)", "Diabetes duration (years)", "Number of antidiabetic medications", 0, 1, 2, 3, 4, "Centre", "Scotland", "Tyneside", "List size", ">5700", "=<5700")
incmean = c('56.2 (6.9)',NA,'62 %','38 %','34.2 (4.3)','137 (15)','58 (11)','8.8 (2.5)','22 (14)','4.3 (1.1)','1.2 (0.3)','1.9 (0.9)','3.0 (1.7)', NA, '23 %','53 %','18 %','5 %','1 %', NA, '77 %','23 %', NA, '49 %','51 %')
ninc = c(146, 146, NA, NA, 146, 146, 146,146, 146,146,146,146,146,146, NA, NA, NA, NA, NA, 146, NA, NA, 146, NA, NA)
excmean = c('53.9 (7.1)', NA, '57 %','43 %','35.0 (4.6)','135 (17)','60 (13)','9.2 (3.2)','24 (15)','4.3 (1.1)','1.1 (0.3)','2.0 (1.4)','3.2 (1.6)', NA, '24 %','44 %','23 %','7 %','3 %', NA, '62 %','38 %', NA, '58 %','42 %')
excn <- c(119,119,NA,NA,119,119,119,118,119,118,118,118,119,119,NA,NA,NA,NA,NA,119,NA,NA,119,NA,NA)
ChiP <- c(0.01,0.54,NA,NA,0.19,0.19,0.15,0.24,0.27,0.90,0.04,0.48,0.37,0.50,NA,NA,NA,NA,NA,0.01,NA,NA,0.20,NA,NA)
mytable = cbind(var, incmean, ninc, excmean, excn, ChiP)
colnames(mytable) <- c("Variable",  "Control mean (SD) or %", "Control N", "Intervention mean (SD) or %", "Intervention N", "P value for difference (Two tailed t-test or Chi2 test)" )
mytable %>% kable(caption = "(ref:caption-DiRECT-participants)", format = "latex", booktabs = TRUE, caption.short = "(ref:scaption-DiRECT-participants)") %>%
  kable_classic_2(full_width = F) %>%
  kable_paper("hover", full_width = F) %>%
   add_indent(c(3, 4, 15:19, 21, 22, 24, 25), level_of_indent = 1) %>%
      kable_styling(full_width = TRUE) %>%
column_spec(1, width = "6cm") %>%
  landscape()
```

### Observational association between BMI change and protein change
Linear regression was used to estimate the effect of BMI change on protein change, where BMI change (endpoint-baseline) was used as the exposure and protein change (endpoint-baseline) was used as the outcome and adjusting for confounders. After adjusting for multiple testing (p<5.8x10^-5^), 254 proteins out of 4601 proteins were associated with BMI change (5.5 % of all protein tested) (Supplementary Table 1). Although overall there was weight loss and a reduction in BMI, beta coefficients can be interpreted as the change in protein level (SDs) per SD increase in BMI. Negative associations were observed with BMI change and scavenger receptor class A member 5 (SCAR5, -0.62 SDs per SD increase in BMI, 95% CI -0.72 to -0.53 p=3.7x10^-29^) and apolipoprotein F (-0.58 SDs per SD increase in BMI, 95% -0.68 to -0.48, p=1.8x10^-24^). BMI change was positively associated with changes in levels of Proto-oncogene tyrosine-protein kinase receptor Ret (RET) (0.59 SDs increase per SD increase in BMI, 95% CI 0.49 to 0.69 p=1.6x10^-25^) and growth hormone receptor (GHR) (0.59 SDs increase per SD increase in BMI, 95% CI 0.49 to 0.69 p=5.8x10^-25^). 

Proteins involved in GO platelet pathways were explored within the results. There were 248 platelet proteins identified in DiRECT. In the linear regression analysis, there was evidence for an effect of BMI change on 31 platelet proteins (12.5%) (Supplementary Table 2). Associations include a positive association between BMI change and collagen type III (0.37 SDs increase per SD increase in BMI, 95% CI 0.26-0.48), p=5.1x10^-10^. There was also a positive association between BMI change and plasminogen activator inhibitor 1 (PAI-1, 0.29 SDs increase per SD increase BMI, 0.17-0.40, p=1.9x10^-6^). The platelet primer insulin-like growth factor 1 (IGF-1) was negatively associated with BMI (-0.28 SDs per SD increase in BMI, 95% CI -0.40 to -0.16, p=4.8x10^-6^). The estimate for the association between BMI change and PAFAH was (-0.16 SDs per SD increase in BMI, 95% CI -0.28 to -0.04, p=9.7x10^-3^) and therefore did not pass multiple testing correction. The estimate for the effect of BMI change on the change in levels was 0.10 SD (95% CI -0.02 to 0.22, p=0.099) and was 0.07 SD for TARC (95% CI -0.04 to 0.19, p=0.22).  

### Association between confounders and exposure/outcomes
There was weak evidence for an association between age and BMI change (0.02 SDs increase in BMI change per 1 year, 95 % CIs -0.0003 to 0.034, p=0.054), therefore indicating less weight loss with increasing age. There was no evidence that sex was associated with BMI change (BMI change 0.073 SDs increase in BMI change in females compared with males, 95 % CI -0.17 to 0.32, p=0.56) (Supplementary Table 2). Age and sex displayed weak associations with protein changes, but these did not pass multiple testing adjustment (Supplementary Tables 3 and 4).

### Association between treatment group and BMI change
Mean BMI change in all participants was -1.8kg/m^2^ (SD 2.5 kg/m^2^), with a mean change in the control group of -0.4 kg/m^2^ (SD 1.3) vs -3.6kg/m^2^ (SD 2.5) in the intervention. The difference in BMI change comparing intervention to control was -3.2 kg/m^2^ (SE 0.25, p=1.52x10^-29^). These analyses suggest that treatment group is a valid instrument for BMI change.

(ref:scaption-box-BMI-change) Box plot of the distribution of BMI change (kg/m^2^) by treatment group
(ref:caption-box-BMI-change) **Box plot of the distribution of BMI change (kg/m^2^) by treatment group**. Median and interquartile ranges displayed by the box. Dotted red line indicates where there is no change in BMI.
```{r box-BMI-change, echo = FALSE, fig.cap="(ref:caption-box-BMI-change)", out.width="85%", fig.align = "center", fig.scap= "(ref:scaption-box-BMI-change)"}
include_graphics(path = "figure/DiRECT/Intervention_BMI_change_boxplot.png", dpi = NA)

```

### Two-stage least squares analysis
Treatment group was used as an instrument for BMI change in a two-stage least squares analysis. After adjustment for multiple testing, BMI change was associated with changes in levels of 171 proteins (3.7 %) (Supplementary Table 5). The protein which showed the strongest association with BMI change was SCAR5 (-0.91 SDs per SD increase in BMI, 95% CI -1.07 to -0.75, p=1.54x10^-23^). Other proteins displayed a positive association with BMI change, including fatty acid-binding protein, heart (FABP1, 0.86 SDs increase per SD increase in BMI, 95% CI 0.69-1.04, p=6.3x10^-19^). Aminoacyclase-1 was also positively associated with BMI change (0.80 SDs increase in protein per SD increase in BMI, 95% CI 0.64 to 0.97, p=4.9x10^-18^). Figure \@ref(fig:qq-plot-DiRECT) shows the qq-plot for the observational associations (A) and instrumental variable two stage least squares analysis (B). The protein with the smallest observed P value in both models is SCAR5, however the other "top hits" are not the same proteins in Figure \@ref(fig:qq-plot-DiRECT).

(ref:scaption-qq) QQ-plots of the expected vs the observed -log~10~(p-values) for the BMI change-protein change effects
(ref:caption-qq) **QQ-plots of the expected vs the observed -log~10~(p-values) for the BMI change-protein change effects**. A) Linear regression B) Instrumental variable two stage least squares analysis.
```{r qq-plot-DiRECT, echo = FALSE, out.width="100%", fig.cap="(ref:caption-qq)", fig.align = "center", fig.scap= "(ref:scaption-qq)"}
include_graphics(path = "figure/DiRECT/QQ_plots_obs_IV.pdf", dpi = NA)
```

In the two stage least squares analysis, there was evidence for an effect of BMI change on 13 platelet proteins. The estimate for the effect of BMI change on collagen type III was 0.53 SDs per SD increase in BMI (95% CI 0.35 to 0.71, p=2.0x10^-8^), larger in magnitude than in the observational analysis (Supplementary Table 2). The estimate for the association between BMI change and the platelet protein, PAFAH, strengthened in the instrumental variable model, where the estimate was -0.42 (95% CI -0.61 to -0.22, p=5.3x10^-5^), thereby passing the multiple testing adjustment. The two stage least squares analysis did not provide evidence for an effect of BMI change on a change in levels of MDC or TARC. 

### Comparison of observational linear regression and TSLS results for the effect of BMI change on protein change
A total of 110 proteins were only associated within the observational linear regression model, whereas 27 proteins were only associated in the TSLS analysis using treatment group as an instrument for BMI change. 144 proteins were associated with BMI in both analyses (\@ref(fig:circos)), with largest effect estimates across methods for SCAR5, Adiponectin, Apolipoprotein F, growth hormone receptor (GHR), receptor-type tyrosine-protein phosphatase U (PTPRU) and sex hormone binding globulin. These can be seen in Figure \@ref(fig:scatter-DiRECT). Of these 144 BMI change-protein change associations that were consistent across both models, 121 were positive, where weight loss led to a reduction in levels of the protein.

(ref:scaption-circos) Circos plot of proteins associated with BMI change in DiRECT
(ref:caption-circos) **Circos plot displaying the protein change (SDs) per SD increase in BMI in DiRECT**. Blue estimates are the two stage least squares estimates and the yellow estimates are from the linear regression. Proteins presented were associated in both models at p=5.8x10^-5^. Figure made using the EpiViz R package https://github.com/mattlee821/EpiViz. 
```{r circos, echo = FALSE, out.width="80%", fig.cap="(ref:caption-circos)", fig.align = "center", fig.scap= "(ref:scaption-circos)"}
include_graphics(path = "figure/DiRECT/circos_plot_BMI_change_protein_ols_tsls.pdf", dpi = NA)
```

\blandscape
(ref:scaption-scatter-DiRECT) Scatter plot comparing proteins association with BMI change across models in DiRECT
(ref:caption-scatter-DiRECT) **Scatter plot displaying the estimate for the change in protein (SDs) per SD increase in BMI in the two stage least squares and linear regression models.** Points are coloured by which models the proteins were associated with BMI change (p<5.8x10^-5^).
```{r scatter-DiRECT, echo = FALSE, out.width="95%", fig.cap="(ref:caption-scatter-DiRECT)", fig.align = "center", fig.scap= "(ref:scaption-scatter-DiRECT)"}
include_graphics(path = "figure/DiRECT/DiRECT_scatter_comparison.pdf", dpi = NA)
```
\elandscape

### Sensitivity analysis
#### Comparison of protein levels at baseline by treatment group
Proteins which were associated with BMI change were compared across control and intervention groups at baseline. This was done as the delta protein variable used in the regression model does not account for absolute levels of protein at baseline, therefore it is possible that the change in levels of protein may appear to be associating with BMI change due to differing basal protein levels across groups. There was not strong evidence for any differences in baseline levels of proteins (p<5.8x10^-5^). However, there was weak evidence that levels of certain proteins, for example apolipoprotein F (beta=-0.20, SE=0.07, p=0.005) and carbohydrate sulfotransferase (beta=0.11, SE=0.04, p=0.006) were different in treatment group compared to control. Figure \@ref(fig:ApoF-carb-boxplot) displays the boxplots for the distribution of levels of these proteins at both baseline and endpoint for the intervention and control group. 

(ref:caption-ApoF-carb-boxplot) **Sensitivity analysis: boxplot of the levels of Apolipoprotein F and Carbohydrate Sulfotransferase at baseline and endpoint, grouped by treatment group.** Median and IQR is indicated by the box. Units are not rank normal transformed.
(ref:scaption-ApoF-carb-boxplot) Sensitivity analysis: boxplot of the levels of Apolipoprotein F and Carbohydrate Sulfotransferase at baseline and endpoint, grouped by treatment group.
```{r ApoF-carb-boxplot, echo = FALSE, out.width="90%", fig.cap= "(ref:caption-ApoF-carb-boxplot)", fig.align = "center", fig.scap= "(ref:scaption-ApoF-carb-boxplot)"}
include_graphics(path = "figure/DiRECT/ApoF_carb_boxplots.jpeg", dpi = NA)
```

#### Using a linear mixed model to estimate effects of treatment group and timepoint on protein levels
To account for individual protein levels at each timepoint, a linear mixed model was used to determine the effect of the intervention on protein levels. The estimate for the interaction between treatment group and timepoint were derived for each protein, which can be interpreted as difference in gradient of protein change comparing the intervention to control. Estimates were compared with the two stage least squares analysis results which provided an estimate for the effect of BMI change on protein change, using the treatment group as an instrument. Units across models are different as the TSLS estimates protein change per SD higher BMI, while the linear mixed model estimates the gradient of the protein comparing endpoint to baseline, therefore the gradient with a reduction in BMI. The estimates from the two models were strongly associated (R^2^ = 0.91, p<2.0x10^-16^, F=49280, intercept=0.017) (Figure \@ref(fig:tsls-lmer)). As a specific example using SCAR5, the estimate from the TSLS was that per SD increase in BMI, SCAR5 reduces by -0.91 SDs in BMI, 95% CI -1.07 to -0.75, p=1.54x10^-23^. The linear mixed model suggests that the estimated difference in gradient of protein change (comparing endpoint to baseline) comparing intervention to control is 0.92 SDs (95% CI 0.75 to 1.09, t=10.8). 

(ref:caption-tsls-lmer) **Scatter plot of the estimates from the TSLS and linear mixed model analyses**. 
(ref:scaption-tsls-lmer) Scatter plot of the estimates from the TSLS and linear mixed model analyses
```{r tsls-lmer, echo = FALSE, out.width="90%", fig.cap= "(ref:caption-tsls-lmer)", fig.align = "center", fig.scap= "(ref:scaption-tsls-lmer)"}
include_graphics(path = "figure/DiRECT/Comparison_TSLS_lmer.pdf", dpi = NA)
```


### Enrichment analysis
To explore disease class enrichment, DAVID Bionformatics 6.8 was used[@Huang2009]. As there was consistent evidence for a BMI effect on 144 proteins across both models in DiRECT, these were compared to the full 4601 protein panel. There was strongest evidence for enrichment in proteins involved in metabolic disease (1.4 fold enrichment, p=4.21x10^-3^), with evidence for enrichment of renal disease (1.8 fold enrichment, p=1.02x10^-2^). CVD had weak evidence for enrichment, but this association did not pass multiple testing correction. 

(ref:caption-DAVID-enrichment-DiRECT) **DiRECT protein enrichment results for disease class using DAVID bioinformatics 6.8 [@Huang2009]**
```{r DAVID-enrichment-DiRECT,  echo=FALSE}
options(knitr.kable.NA = "")
Term = c('Metabolic', 'Renal', 'Neurological', 'Psych', 'Pharmacogenomic', 'Cancer', 'Reproduction', 'Normal variation', 'Aging', 'Cardiovascular')
Count = c(73,34,49,36,51,56,25,16,21,56)
Percentage = c(53,24,35,26,37,40,18,12,15,40)
List <- c(118,118,118,118,118,118,118,118,118,118 )
Pophits <- c(1451,518,884,589,964,1093,381,209,324,1214)
Poptotal <- c(3204, 3204,3204,3204,3204,3204,3204,3204,3204,3204)
FE <- c(1.4,1.8,1.5,1.7,1.4,1.4,1.8,2.1,1.8,1.2)
Pval <- c('2.34E-04','5.72E-04','9.61E-04','1.38E-03','2.11E-03','2.23E-03','4.49E-03','8.09E-03','1.21E-02','2.51E-02')
Bonf_adj_P <- c('4.21E-03','1.02E-02','1.72E-02','2.45E-02','3.74E-02','3.94E-02','7.79E-02','1.36E-01','1.96E-01','3.68E-01')
mytable = data.frame(cbind(Term, Count, Percentage, List, Pophits, Poptotal, FE, Pval, Bonf_adj_P))
colnames(mytable) <- c('Term (Genetic Association Database disease class)',	'Count',	'%',	'List Total','	Population Hits',	'Population Total',	'Fold Enrichment',	'P Value',	'Bonferroni-adjusted P value' )
mytable %>% kable(caption = "(ref:caption-DAVID-enrichment-DiRECT)", booktabs = TRUE) %>% 
  kable_classic_2(full_width = F) %>%
 kable_styling(full_width = TRUE) %>%
  column_spec(1, width = "5cm")   %>%
  landscape()
```

Using StringDB and inputting p values for the association between BMI change and each protein change, there was evidence for enrichment of KEGG pathways. Using the UniProt IDs and p-values from the linear regression analysis for BMI change and protein change, there was evidence that proteins altered by protein change in the linear regression model were involved in cytokine-cytokine receptor interaction (enrichment score = 0.14, pathway size = 282, p = 2.4x10^-4^) and complement and coagulation cascades (enrichment score = 0.36, pathway size = 82, p = 9.8x10^-4^). These were the only pathways associated with the "top of the input", therefore indicating enrichment of proteins most strongly associated with BMI change. There was also enrichment for platelet activation, however the analysis suggested that this enrichment was at the bottom of the protein list (enrichment score 4.9, pathway size 122, p=6.5x10^-5^), therefore more proteins involved in platelet activation were not associated with BMI change. The network of proteins involved in platelet activation is shown in Figure \@ref(fig:platelet-enrichment). KEGG enrichment results were not available by inputting the ranked p values for the proteins from the instrumental variable analysis.

\blandscape
\thispagestyle{empty}

(ref:caption-platelet-enrichment) **Network of proteins in DiRECT which are involved in platelet activation**. Nodes represent individual proteins. The halo colour represents the rank of the protein where blue indicates the top of the list (most strongly associated with BMI change) and red indicates the bottom of the list. The edges indicate known interactions between proteins. Figure from made using string-db.org

(ref:scaption-platelet-enrichment) Network of proteins in DiRECT which are involved in platelet activation. 

```{r platelet-enrichment, echo = FALSE, out.width="75%", fig.cap= "(ref:caption-platelet-enrichment)", fig.align = "center", fig.scap= "(ref:scaption-platelet-enrichment)"}
include_graphics(path = "figure/DiRECT/string_platelet_activation_network.png", dpi = NA)
```
\elandscape

## Discussion

In this chapter, a clinical trial implementing caloric restriction-induced weight loss was utilised as a second line of evidence to assess the effect of BMI on the plasma proteome. This analysis provided evidence for a broad effect of weight loss on the plasma proteome, providing results the effect of caloric restriction on the most extensive set of proteins to date. 

The current study identified 144 BMI-related protein changes that were consistent across both models. This includes a reduction in BMI reducing levels of GHR, while increasing levels of apolipoprotein F, SHBG and IGFBP1/2. Table \@ref(tab:weight-loss-protein) provides a summary of findings from the literature from previous studies which have explored the effect of caloric restriction on plasma proteins. As detailed in the table, previous studies have mainly used mass spectrometry. One study used SomaLogic[@Carayol2017], however this panel only included 1129 proteins. The current study utilised over fourfold more proteins, therefore providing more in depth readouts of biological processes. Findings are in concordance with previous studies[@Geyer2016; @Figarska2020; @OllerMoreno2018; @Piening2018; @Carayol2017; @Bruderer2019]. These protein changes replicate across cohorts and proteomic platforms and are more likely to be robust associations. Proteins altered with caloric restriction have been implicated in disease including coronary artery disease and T2D within the same cohort[@Ritchie2019]. A mediation analysis suggested that proteins mediate the risk between a T2D genetic risk score (GRS) and incidence of T2D, such as IGFBP1/2, SHBG and GHR. Looking at individual proteins, a SNP which increases levels of ApoF also increases the risk of myocardial infarction and coronary heart disease[@Liu2021], therefore this apolipoprotein could contribute to the increased risk of thrombotic events seen with a higher BMI.

(ref:caption-weight-loss-protein) **Summary of current literature on the effect of weight loss on the plasma proteome**
(ref:scaption-weight-loss-protein) Summary of current literature on the effect of weight loss on the plasma proteome
```{r weight-loss-protein,  echo=FALSE}
options(knitr.kable.NA = "")
paper = c('Proteomics reveals the effects of sustained weight loss on the human plasma proteome','Proteomic profiles before and during weight loss: Results from randomized trial of dietary intervention','The differential plasma proteome of obese and overweight individuals undergoing a nutritional weight loss and maintenance intervention','Integrative Personal Omics Profiles during Periods of Weight Gain and Loss','Protein quantitative trait locus study in obesity during weight-loss identifies a leptin regulator','Analysis of 1508 plasma samples by capillary-flow data-independent acquisition profiles proteomics of weight loss and maintenance')
author = c('Geyer et al., 2016','Figarska et al., 2020','Oller-Moreno et al., 2018','Piening et al., 2018','Carayol et al., 2017','Bruderer et al., 2019')
platform = c('Mass spectrometry','Olink CVD II, CVD III and inflammation','Mass spectrometry','Luminex and ProSeek by Olink','SomaLogic','Mass spectrometry')
findings = c('Participants: 43 individuals with obesity before and after 8-week weight loss.
Total proteins associated: 93/1294,
Overlapping findings: Weight loss leads to increase in ApoF, ITIH3, SHBG, corticosteroid-binding globulin.',
'Participants: 609 from the Diet Intervention Examining The Factors Interacting with Treatment Success (DIETFITS) study.
Total proteins associated: 130/263 associated with change in BMI.
Overlapping findings: Changes in IGFBP1/IGFBP2, E-selectin, CD163.',
'Participants: 473 in the nutritional programme DiOGenes.
Total proteins associated: 39/183.
Overlapping findings: Changes in SHBG, Adiponectin, IGFBP2, galectin-3-binding protein.', 
'Participants: 23 with overweight or obesity undergoing weight loss.
Total proteins associated: 27/276.
Overlapping findings: changes in leptin.',
'Participants: 494 participants on low DiOGenes on low calorie diet.
Total proteins associated: 104/1129.
Overlapping findings: Changes in leptin, GHR, TIG2, IGFBP2, RET proto-oncogene (granulysin).',
'Participants: 477 in DiOGenes.
Total proteins associated: 271/465.
Overlapping findings: Changes in Apolipoprotein F, INHBC, SHBG.')
mytable = cbind(paper, author, platform, findings)
colnames(mytable) <- c("Paper", "Author, year", "Proteomic platform", "Study summary")
mytable %>% kable(caption = "(ref:caption-weight-loss-protein)", booktabs = TRUE, caption.short = "(ref:scaption-weight-loss-protein)") %>% 
  kable_classic_2(full_width = F) %>%
  #kable_paper("hover", full_width = F)
 kable_styling(full_width = TRUE) %>%
column_spec(4, width = "8cm") %>%
    column_spec(1, width = "7cm") %>%
  landscape()
```

This study also provides some novel proteins associated with a reduction in BMI including SCAR5, chondroadherin (CHAD), fumarylacetoacetase (FAAA) and  aminoacylase-1. A reduction in BMI led to an increase in plasma levels of SCAR5. This protein is involved in mediating commitment of mesenchymal stem cells into adipocytes and is highly expressed in white adipose tissue (WAT)[@Lee2017a]. A reduction in BMI also increased levels of chondroadherin (CHAD). Chondroadherin is a protein which is important in regulating exctracellular matrix organisation in cartilage and bones[@Hessle2013; @Iozzo2015]. CHAD knockout mice show decreased bone strength compared with wildtype[@Hessle2013]. This suggests that weight reduction may be beneficial for bone strength. The function of these proteins and relationship with adiposity needs to be further characterised.

The effect of BMI on specific platelet-related proteins was explored by extracting the estimates for proteins involved in platelet GO pathways. Both analyses indicated that a reduction in BMI reduces the levels of collagen type III. Collagen activates platelets by binding to receptors such as the GPVI receptor, therefore this reduction may reduce platelet activation[@Maurice2006]. The instrumental variable analysis detected an effect of BMI change on PAFAH. Here, a reduction in BMI led to higher levels of PAFAH. This platelet protein was detected in Chapter \@ref(BMI-protein-MR). PAFAH is protein which has been shown to bind to lipoproteins, thereby reducing their oxidation[@Noto2003]. The comparison across studies will be discussed further in Chapter \@ref(Comparison-proteome). The chemokines utilised in Chapter \@ref(chemokine-platelets), MDC and TARC, were included in the SomaLogic panel, therefore were also specifically looked up within the results. The evidence suggested that a reduction in BMI did not alter levels of MDC and TARC, however it is also possible that the sample size was limited to detect smaller magnitudes of effects.

The enrichment results comparing the 144 proteins which were altered with BMI change to the full 4601 proteins tested confirmed this as there was strongest evidence for enrichment of proteins involved in metabolic disease and renal disease, however the effect on CVD did not pass multiple testing adjustment. Pathway enrichment using StringDB also suggested that proteins most altered with BMI change were involved in cytokine-cytokine receptor interactions and complement and coagulation cascades, rather that platelet activation. It could be that the effect on inflammation and coagulation may be more immediate effects, and that changes in platelet activation may either require a more sustained weight loss, or a further reduction in BMI to see effects.

Limitations from the current analysis therefore include possible protein effects that are not caused by a change in BMI. Despite the use of the treatment group as an instrument in a two stage least squares analysis, the treatment group was also associated with an alteration in other factors: those who were allocated TDR also were taken off any T2D medication and were more likely to achieve remission of T2D. It is possible that the analysis is therefore picking up protein effects that are caused by a change in medication or T2D remission, rather than as a result in loss of fat. Participants would have also lost muscle mass throughout the intervention[@Santanasto2011], therefore it is not certain that effects are attributable to loss of fat. Despite this, there is evidence that in a group of participants with obesity, the majority of weight lost is fat mass[@Backx2016]. Thirdly, BMI decreased by an average of 0.4kg/m^2^ in the control group and by 3.6 kg/m^2^ in the intervention group. As participants started at a mean BMI of 34-35 kg/m^2^, the majority of participants will still have obesity even after the intervention. This may limit protein effects observed, as a certain BMI may need to be reached to see an alteration in protein levels (i.e. the relationship may not be linear). Finally, it is unclear how generalisable results would be to the wider population. It is often the case with diets and caloric restriction that they only result in weight loss short-term and this is likely because the body adapts to the lower intake of calories. Therefore the changes which are seen in levels of proteins may not be so dynamic. The use of bariatric surgery may help to ensure weight loss and the associated changes are sustained.

Overall, this study provided evidence that weight loss leads to a substantial change in the composition of plasma proteins, replicating effects seen previously and providing some novel associations. Proteins altered with higher BMI in a Mendelian randomisation framework can be reversed with weight loss. Proteins altered with weight loss were enriched for renal disease and metabolic disease. 
