---
output: 
    thesisdown::thesis_pdf: default
bibliography: bib/library.bib
csl: csl/nature.csl
space_betwee_paragraphs: true
fig_caption: true
always_allow_html: yes
link-citations: true
toc-depth: 2
lot: true
lof: true
indent: true
header-includes: # include other LaTeX packages here
    \usepackage{booktabs}
    \usepackage{longtable}
    \usepackage{siunitx}
    \usepackage[left]{lineno}
    \usepackage{float}
    \floatplacement{figure}{H}
    \linenumbers
    \pagestyle{plain}
    \raggedbottom 
    \usepackage{indentfirst}
editor_options: 
  chunk_output_type: console
---

```{r, echo=FALSE, message=FALSE}
source("~/OneDrive - University of Bristol/PhD/Main project/Thesis/index/packages.R")
```
# Effects of adiposity on the human plasma proteome: Observational and Mendelian randomization estimates {#BMI-protein-MR}

The work within this chapter has been published in the International Journal of Obesity [@Goudswaard2021]. I am the first author of the publication; I performed all analyses, created all figures and tables and wrote the manuscript. 

Supplementary Tables 1-7 can be found here https://github.com/lucygoudswaard/mythesis/blob/master/index/figure/BMI_protein_INTERVAL/BMI_protein_supplementary_Tables.xlsx 

## Introduction

Obesity has tripled worldwide since 1975, now affecting around 40% of adults in the United States and 26% of adults in the UK [@Bluher2019]. The average body mass index (BMI) of the UK adult population is now in the conventional 'overweight' category (BMI between 25 and 30 kg/m^2^) [@Wade2018] and 'overweight' is now more common than 'normal-weight' in middle age in many high-income countries [@NCD-RisC2016]. BMI is often used as a proxy for adiposity given high correlations between BMI and more objectively measured fat mass indices [@Bell2018a]. Higher adiposity is a major risk factor for various noncommunicable diseases including cardiovascular diseases such as coronary heart disease (CHD) and stroke [@Nordestgaard2012; @Dale2017; @Wolk2003a]. These BMI-disease associations are supported by prospective observational studies and, more recently, by Mendelian randomization (MR) studies [@Nordestgaard2012; @Dale2017] which use genetic variation reliably associated with BMI to re-estimate effects of BMI on disease outcomes. Given the properties of genetic variation, this method helps to overcome issues such as confounding and reverse causation which commonly occur with observational studies [@Smith2003].

Despite MR studies supporting a causal role of adiposity for cardiometabolic diseases, and randomised trials supporting the effectiveness of weight loss in reducing disease risk [@Wing2011], the molecular footprint of adiposity is not well understood. Previous studies have largely focused on the impact of higher BMI on the lipidome including traits such as cholesterol and triglycerides in lipoprotein subtypes (e.g. low-density lipoprotein (LDL)  and high-density lipoprotein (HDL) particles) [@Bell2018a; @Wurtz2014] and on inflammatory molecules such as C-reactive protein (CRP) [@Wurtz2014; @Timpson2011]. 

A benefit of studying the systematic effects of BMI on the circulating proteome is that proteins are often more suitable pharmacological targets than metabolites. Efforts to study the effect of BMI on the proteome have generally been in an observational framework [@Cominetti2018]. It is estimated that 25% of proteins in the human proteome circulate in blood [@Gold2012], which is important as the majority of druggable targets are such proteins [@Imming2006]. Studying the effect of BMI on a large set of proteins has only recently become possible with newly developed proteomic technologies such as the SomaLogic platform, with the ability to quantify enzymes, protein kinases and transport proteins with unprecedented sensitivity [@Rohloff2014]. Utilisation of SomaLogic within a trial or cohort setting has recently become more widespread, such as within the INTERVAL study, a UK cohort of blood donors [@Sun2018]. There is evidence that proteins which change as a result of a higher BMI may contribute to cardiometabolic disease: identification of such proteins is important in understanding how higher BMI causes disease and to identify targets which may benefit from pharmacological intervention. It is possible that proteins that are altered by BMI may contribute to cardiometabolic disease by activating platelets (Chapter \@ref(chemokine-platelets)).

In this chapter, the aim was to measure associations between adiposity and the human proteome and to also estimate the underlying effects in a causal framework (Figure \@ref(fig:BMI-protein-graphic)). Using data on 2737 participants from INTERVAL, the effects of BMI on 4034 (3622 unique) plasma protein traits were estimated, in both observational and MR frameworks. The agreement between effect estimates from different methods were examined and enrichment analyses of the most strongly altered proteins were performed to map their potential relevance to cardiovascular disease.

```{r BMI-protein-graphic, echo = FALSE , fig.scap="Schematic of the associations explored in chapter", fig.cap="Schematic of the associations explored: this chapter explores the causal effect of BMI on the plasma proteome and explores whether these changes might contribute to cardiovascular diseases. Figure made using BioRender.com.", out.width="90%"}
include_graphics(path = "figure/BMI_protein_INTERVAL/Thesis_graphic_overview_BMI_protein_CVD.png", dpi = NA)
```

## Methods

### Study population

The present study was conducted on a random subset of participants from INTERVAL who had basic phenotype data and plasma proteins measured by SomaLogic. This included up to 2737 participants mostly of European descent across analyses described below. More details of this study can be found in Chapter \@ref(common-methods).

### Measurement of circulating proteins

Plasma proteins were measured in INTERVAL participants at baseline (before randomisation of assignment to the time interval between blood donation) using the SomaScanÂ® by SomaLogic [@Sun2018]. This platform uses 4034 modified nucleotides known as Slow Off-rate Modified Aptamers (SOMAmers) which make direct contact with proteins, enabling detection of 3622 unique proteins or protein complexes and quantifies them using a DNA microarray [@Rohloff2014]. Separate SOMAmers can bind to isoforms of the same protein, but can also bind to the same protein at different sites (which can be impacted by post-translational modifications or complexes formed with other proteins). We therefore have included all 4034 SOMAmers. The extensive number of proteins measured, with no missingness and in a cohort of 2737 participants, provides a rich proteomic dataset. The proteins were measured in relative fluorescence units (RFUs) and quality control (QC) was performed as described by Sun et al. [@Sun2018]. There was no missingness across protein variables. The proteomic data used had been pre-adjusted (using linear regression) for age, sex, duration between blood draw and sample processing (1 day or less vs >1 day), and the first three genetic principal components, with the residuals inverse normal rank transformed. All following analyses use this 'pre-adjusted' data as input.

### Statistical analyses

The population characteristics of INTERVAL participants with SomaLogic data who were included in this study (N range: 2422 to 2737 due to missing data for covariables) were compared to those INTERVAL participants who were not included (N range: 27174 to 30721) to assess generalisability of any BMI-protein associations to the wider INTERVAL sample. Population characteristics evaluated were age, sex, weight, height, BMI, smoking frequency, and alcohol intake. Differences in population characteristics among the two INTERVAL sub-sets were tested by a two-sided t-test for continuous traits and a two-sided Chi-square test for categorical variables. Observational analyses were conducted using linear regression to examine associations between BMI (in normalized standard deviation (SD) units based on a rank normal transformation (rntransform() from "moosefun" package https://github.com/hughesevoanth/moosefun) and each standardised protein trait as a dependent (outcome) variable. Two linear models were used (using lm() function from R "stats" package): 1) adjusted for age and sex and 2) additionally adjusted for smoking and alcohol consumption (each as an ordered categorical variable). Given that the procedure which generates the "pre-adjusted" data (adjustment for covariables before rank normal transformation of proteins) can reintroduce correlations [@Pain2018], age and sex are again used as covariables here. The estimates derived from models 1) and 2) therefore reflect the normalized SD-unit difference in each protein trait per normalized SD-unit (4.8 kg/m^2^) higher BMI. Associations of covariables with BMI and protein traits were also examined using linear regression.

A Shapiro-Wilk test was used to confirm whether the GRS showed a normal distribution. MR analyses were conducted using two-stage least squares (2SLS) regression models with robust standard errors, using the systemfit function as part of the systemfit package [@Henningsen2007], with measured BMI in SD units and the GRS for BMI as the instrumental variable. These MR estimates reflect the normalized SD-unit difference in each protein trait per normalized SD-unit (4.8 kg/m^2^) higher BMI. We report estimates from the direct linear associations between BMI and proteins as "observational" estimates and those from the 2SLS causal effect estimates as "MR estimates". Agreement between observational and MR estimates was examined using a separate linear regression. This was performed: 1) for all proteins, and 2) excluding the proteins that fell below our P-value reference point for strong evidence (defined below) to examine whether agreement is limited to "top hits" or applies throughout the effect distribution. Agreement between observational estimates and MR estimates would suggest that there are causal effects of BMI across the general proteome, with differences in estimates suggesting confounding of observational estimates. 

To account for multiple testing, a Bonferroni correction was used to adjust results. This was informed by the correlation between proteins, adjusting only for the estimated number of independent traits (Figure \@ref(fig:Corr-mat-proteins)). Correlation was assessed by a Spearmanâs correlation matrix. From a starting number of 4034, the number of independent proteins was  3655 (using a correlation cut-off of r  = 0.8 or tree cut height = 0.2 between proteins, Figure \@ref(fig:Dend-proteins)), dendrogram made using  "iPVs" package https://github.com/hughesevoanth/iPVs). We utilised a Bonferroni adjusted P-value of 0.05/3655 = 1.4x10^-5^ to indicate strong evidence in this sample. Full results are presented in the supplementary material. 

```{r Corr-mat-proteins, echo = FALSE, out.width = "90%",fig.cap="Correlation matrix of all protein traits analysed by SomaLogic (4,034) where the colour corresponds to the correlation coefficient ranging from 1 (dark blue) to -1 (dark red)", fig.scap="Correlation matrix of protein traits analysec by SomaLogic", fig.scap="Correlation matrix of all protein traits analysed by SomaLogic"}
include_graphics(path = "figure/BMI_protein_INTERVAL/Correlation_matrix_proteins.pdf", dpi = NA)

```

\blandscape
```{r Dend-proteins, echo = FALSE, out.width="95%", fig.cap="Cluster dendrogram showing the hierarchical relationship between protein levels. Height is calculated as (1-correlation coefficient). At a height of 0.2 and 0.4 (red dashed lines), the number of independent proteins was 3,655 and 3,016 respectively.", fig.scap = "Cluster dendrogram showing the hierarchical relationship between protein levels.", fig.scap="Cluster dendrogram showing the hierarchical relationship between protein levels"}
include_graphics(path = "figure/BMI_protein_INTERVAL/dendrogram.pdf", dpi = NA)
```
\elandscape

### Enrichment analysis
To investigate whether any protein groups showed a particularly strong relationship with BMI and disease (signal detection), protein features were clustered for further analysis. First, a principal component analysis (PCA; prcomp() function from the R "stats" package) on the proteins, not the individuals, was performed on the "pre-adjusted" (see above) dataset (Figure \@ref(fig:PCs)A-D). The top "n" PC eigenvectors, as identified by a scree plot of the PCA eigenvalues (Figure \@ref(fig:PCs)E), were carried forward into an unsupervised k-means analysis (kmeans() function from the R "stats" package). Nineteen k-means analyses were run altering the value of k (number of clusters) from two to 20. To identify an appropriate number of protein clusters (k) we generated a scree like plot (Figure \@ref(fig:PCs)F). Here we plotted the variance explained by clusters, for each k, as estimated as the sum of squares explained by clusters (betweenness) over the total sums of squares, and looked for the smallest k with the maximum variance explained (a plateau). In summary, we used a data reduction method (PCA) to identify major axes (PCs) of the protein data that were then utilised in a machine learning clustering algorithm (k-means) to identify clusters of proteins that share abundance similarities across individuals.

```{r PCs, echo = FALSE, out.width = "95%", fig.cap="Principal component analysis (PCA) and k-means clustering of proteins provide evidence for five clusters. A-D) Principal component (PC) 1 vs PC2-PC5 for the study protein data. Each dot represents a protein, and the colors represent the clusters identified by the k-means analysis (1=black, 2=red, 3=green, 4=dark blue, 5=light blue, 6=pink). E) PC scree plot displaying the proportion of variance explained by each of the first 20 PCs. F) K-means scree like plot displaying the variance explained by clusters for each of the 19 k-means analysis, where in each analysis k (the number of clusters) was set from two to 20.", fig.scap = "Principal component analysis (PCA) and k-means clustering of proteins"}
 include_graphics(path = "figure/BMI_protein_INTERVAL/PCs2to5v1.pdf", dpi = NA)
 
```

To explore whether there was a systematic difference in the association of proteins within these clusters and BMI, the beta coefficients from the observational linear regressions or MR models were transformed into their absolute values and divided by their standard error (SE). The absolute betas divided by their SEs in each cluster was compared using a one-tailed pairwise Wilcox test to identify which clusters showed a stronger association with BMI. For the cluster(s) showing evidence for larger absolute beta coefficients, an enrichment analysis was performed using DAVID bioinformatics resources 6.8 [@Huang2009]. Enrichment was assessed by using the uniprot IDs for the proteins in the cluster and comparing these proteins with the uniprot IDs of the full SomaLogic protein list. Enrichment for protein involvement in disease (using the genetic association database (GAD) disease classes [@Becker2004a]) of the protein cluster was assessed by fold enrichment and a Bonferroni-corrected P-value to account for multiple testing. Proteins that were associated with BMI in confounder-adjusted observational analyses at P<1.4x10^-5^ were also entered into the disease enrichment tool and compared with the total proteins (as described for cluster enrichment). Analyses were performed using R version 3.4.2 [@Team2019a]. R code used for analyses available upon request.


## Results

### Participant characteristics
INTERVAL participants included in this study (those with proteomic data), had a mean age of 45.0 years (SD of 14.1 years) and 48.3% were female (Table \@ref(tab:Inc-excl-part)). Mean BMI was 25.9 kg/m^2^ (SD of 4.8 kg/m^2^) and the majority of participants were non-smokers (59.1%). Nearly a quarter (23.5%) reported currently or previously smoking daily and 71.5% reported drinking alcohol at least once a week. Participants with proteomic data were representative of the full INTERVAL cohort (Table \@ref(tab:Inc-excl-part)).

```{r Inc-excl-part,  echo=FALSE}
options(knitr.kable.NA = "")
var = c("Age", "Sex", "Male", "Female", "Weight (kg)", "Height (cm)", "Body mass index (kg/m2)", "Smoking frequency", "Never", "Occasional", "Most days or every day", "Alcohol intake frequency", "Rarely", "Less than once a week", "One or two times a week", "Three to five times a week / most days")
incmean = c('45.0 (14.1)', NA, '51.70%','48.30%','78.5 (16.1)','172.7 (9.7)','25.9 (4.8)', NA, '59.10%','11.90%','29.00%', NA, '11.70%','16.90%','37.40%','34.10%')
ninc = c(2737, 2737, NA, NA, 2736, 2730, 2729, 2675, NA, NA, NA, 2422, NA, NA, NA, NA)
excmean = c('44.9 (14.2)', NA, '49.70%','50.30%','78.6 (16.0)','172.2 (9.6)','26.4 (4.7)', NA, '59.40%','12.00%','28.50%', NA,'12.60%','17.10%','37.70%','32.50%')
excn <- c(30721, 30721, NA, NA, 30719, 30661, 30659, 29958, NA, NA, NA, 27124, NA, NA, NA, NA)
ChiP <- c(0.76, 0.04, NA, NA, 0.26, 0.005, 0.66, 0.88, NA, NA, NA, 0.34, NA, NA, NA, NA)
mytable = cbind(var, incmean, ninc, excmean, excn, ChiP)
colnames(mytable) <- c("Variable",  "Included Mean (SD) or %", "Included N", "Excluded Mean (SD) or %", "Excluded N", "P value for difference (Two tailed t-test or Chi2 test)" )
mytable %>% kable(caption = "Comparison of included vs excluded INTERVAL participants", booktabs = TRUE) %>% 
  kable_classic_2(full_width = F) %>%
  #kable_paper("hover", full_width = F)
   add_indent(c(3, 4, 9, 10, 11, 13, 14, 15, 16), level_of_indent = 1) %>%
 kable_styling(full_width = TRUE) %>%
column_spec(1, width = "6cm") %>%
  landscape()
```

### Observational estimates of associations of BMI with protein traits 

In a linear regression model adjusted for age and sex among 2729 adults, BMI (per SD higher) was associated with 1576 proteins (39%) at the level P < 1.4 x 10^-5^ (multiple testing reference point, Supplementary Table 1). In a second model additionally adjusting for frequencies of smoking and alcohol intake among 2 380 adults, there were 1447 associations at the same reference point (Supplementary Table 2). The strongest positive associations were with leptin (0.74 SD, 95% CI 0.71-0.77, P=9.9x10^-324^) and adipocyte fatty acid binding protein (FABP4) (0.58 SD, 95% CI 0.55-0.62, P=6.4x10^-211^). BMI (per SD) was also strongly positively associated with inflammatory proteins such as Complement Factor I (0.46 SD, 95% CI 0.43-0.50, P=5.6x10^-122^) and CRP (0.44 SD, 95% CI 0.41-0.48, P=8.2x10^-112^). BMI (per SD) also showed strong negative associations with proteins such as insulin-like growth factor-binding protein (IGFBP) 2 (-0.48 SD, 95% CI -0.51 to -0.44, P=2.7x10^-133^) and sex hormone-binding globulin (SHBG) (-0.43 SD, 95% CI -0.47 to -0.39, P=2.4x10^-106^).

### Observational associations of covariables with BMI and protein traits

Age, sex, and frequencies of smoking and alcohol intake were each associated with BMI (Table \@ref(tab:INT-confounders-BMI)). Males had a higher BMI than females (0.17 SD, 95% CI 0.10-0.25, P=5.8x10^-6^). Age was positively associated with BMI (0.01 SD higher per year older, 95% CI 0.009-0.015, P=1.2x10^-18^). Smoking frequency was positively associated with BMI, but alcohol intake frequency was negatively associated with BMI. Covariables (age, sex, smoking and alcohol) showed associations with protein traits (Supplementary Tables 3-6 and Figures \@ref(fig:QQ-confounder-proteins) A-D). There was evidence for 18 associations between age and proteins, 26 associations between sex and proteins, 38 proteins associated with smoking and 137 proteins associated with alcohol at the Bonferroni-adjusted level of p<1.4x10^-5^.

```{r INT-confounders-BMI,  echo=FALSE}
options(knitr.kable.NA = "")
V = c( "Age (years)", "Sex (1=female, 2=male)", "Smoking frequency (1=never, 2=occasinal, 3=most days/ every day)", "Alcohol intake frequency (1=rarely, 2=<1 a week, 3=1-2 times a week, 4=most days/every day)")
N = c(2729, 2729, 2667, 2415)
B = c(0.012, 0.17, 0.022, 0.021)
SE <- c(0.001, 0.038, 0.022, 0.021)
P <- c(1.23E-18, 5.80E-6, 3.16E-5, 7.08E-5)
AdjR <- c(0.028, 0.007, 0.006, 0.006)
Fstat <- c(78.8, 20.6, 17.4, 15.8)
CI <- c(0.003, 0.075, 0.043, 0.041)
LowerCI <- c(0.009, 0.10, 0.05, -0.12)
UpperCI <- c(0.015, 0.25, 0.13, -0.04)
mytable = data.frame(cbind(V, N, B, SE, P, AdjR, Fstat, CI, LowerCI, UpperCI))
colnames(mytable) <- c('Variable',	'N',	'Beta coefficient per 1-unit increase in confounder (SDs)',	'Standard error',	'P value',	'Adjusted R2',	'F statistic' , "95% CI", "Lower 95% CI", "Upper 95% CI")
mytable %>% kable(caption = "Associations between covariables (exposure) and standardised BMI (outcome)", booktabs = TRUE) %>% 
  kable_classic_2(full_width = F) %>%
 kable_styling(full_width = TRUE) %>%
column_spec(1, width = "4cm")   %>%
  column_spec(3, width = "2cm")   %>%
  landscape()
```

```{r QQ-confounder-proteins, echo = FALSE, out.width = "95%", fig.cap="Q-Q plots for the expected against observed -log10(pvalues) of the association of age (A), sex (B), smoking (C) and alcohol (D) with protein traits", fig.scap="Q-Q plots of the expected against observed -log10(pvalues) for the associations between covariables and protein traits"}
include_graphics(path = "figure/BMI_protein_INTERVAL/QQ_confounder_proteins.pdf", dpi = NA)

```

### Associations of the GRS for BMI with measured BMI and covariables

The distribution of the GRS among participants was normal (mean=0.08, SD=0.29, W=0.99, P=0.73, N=2729). The GRS was associated with BMI, explaining 2.8% of its variance (R^2^=0.028, P=1.6x10-18, Table \@ref(tab:INT-GRS-confounders)). There was no strong evidence of association between GRS and age (R^2^=0.001, P=0.11), sex (R^2^=6x10-5, P=0.28), smoking frequency (R^2^=< 0.0001, P=0.91), or alcohol intake (R^2^<0.0001, P=0.44).

```{r INT-GRS-confounders,  echo=FALSE}
options(knitr.kable.NA = "")
V = c("BMI (SDs)", "BMI (kg/m2)", "Age", "Sex (1=female, 2=male)", "Smoking frequency (1=never, 2=occasinal, 3=most days/ every day)", "Alcohol intake frequency (1=rarely, 2=<1 a week, 3=1-2 times a week, 4=most days/every day)")
N = c(2729, 2729, 2737, 2737, 2675, 2422)
B = c(0.57, 2.54, -1.47, -0.04, -0.02, -0.06)
SE <- c(0.06, 0.31, 0.92, 0.03, 0.14, 0.07)
P <- c(1.64E-18, 4.82E-16, 0.11, 0.28, 0.91, 0.44)
AdjR <- c(0.028, 0.024, 0.001, 5.96E-05, -0.0004, -0.0002)
Fstat <- c(78.20, 66.68, 2.55, 1.16, 0.01, 0.60)
mytable = data.frame(cbind(V, N, B, SE, P, AdjR, Fstat))
colnames(mytable) <- c('Variable',	'N',	'Beta coefficient (per 1-unit increase in GRS)',	'Standard error',	'P value',	'Adjusted R2',	'F statistic' )
mytable %>% kable(caption = "Associations of the genetic risk score for BMI with reported BMI and covariables", booktabs = TRUE) %>% 
  kable_classic_2(full_width = F) %>%
 kable_styling(full_width = TRUE) %>%
column_spec(1, width = "5cm")   %>%
  landscape()
```

### MR estimates of associations between BMI and protein traits

In MR analyses, eight unique BMI-protein associations were detected at the level P<1.4x10^-5^ (multiple testing reference point, Figure \@ref(fig:ObsMRforestplot2)). MR estimates provide an estimate of the causal association between protein (in SDs) per SD higher BMI. The strongest association of BMI was again with leptin (0.63 SD, 95% CI = 0.48-0.79; P=1.6x10^-15^); this was followed by the association with FABP4 (0.65 SD, 95% CI = 0.46-0.83; P=6.7x10^-12^). A strong negative association was also seen between BMI (per SD) and SHBG (-0.45 SD, 95% CI -0.65 to -0.25, P=1.4x10^-5^). Other BMI-protein associations (P<1.4x10^-5^) included positive associations with fumarylacetoacetase (FAAA), inhibin beta C chain and complement C5, and negative associations with receptor-type tyrosine-protein phosphatase delta and PILR alpha-associated neural protein. Supplementary Table 7 provides the full MR results.

```{r ObsMRforestplot2, echo = FALSE, out.width = "95%", fig.cap="Forest plot of the strongest BMI-protein Mendelian randomisation estimates and the corresponding observational estimates"}
include_graphics(path = "figure/BMI_protein_INTERVAL/MR_obs_compar_forestplot.pdf", dpi = NA)

```

### Comparison of observational and MR estimates

The distribution of P-values for associations between BMI and protein traits suggested an overrepresentation of signal for the observational estimates of BMI and protein traits; far more than expected from chance alone (Figure \@ref(fig:QQ-obs-MR)A). In contrast to this, the extent of this overrepresentation was reduced considerably in the MR (Figure \@ref(fig:QQ-obs-MR)B).

```{r QQ-obs-MR, echo = FALSE, out.width = "95%", fig.cap="A) Q-Q plot of expected against observed -log10(pvalues) for the unadjusted observational BMI-protein trait estimates B) Q-Q plot of expected against observed -log10(p) values for the Mendelian Randomization BMI-protein trait estimates.", fig.scap = "Q-Q plot of the expected against observed -log10(pvalues) for BMI-protein estimates"}
include_graphics(path = "figure/BMI_protein_INTERVAL/QQ_obs_MR.pdf", dpi = NA)

```

The unadjusted and confounder-adjusted regression coefficients for BMI and protein traits were strongly associated (Beta=0.99 SDs, R^2^=0.99, P=9.9x10^-324^, Figure \@ref(fig:Estimate-comparison-obs)). Compared with the observational estimates, the MR estimates were less precise, but there was a strong positive association between the beta coefficients from observational and MR estimates (Beta=0.68 SDs, R^2^=0.33, P=9.9x10^-324^) (Figure \@ref(fig:Estimate-comparison-obs-mr)). After removing the proteins where P<1.4x10^-5^, the strength of association between unadjusted and adjusted observational estimates remained, but the association between observational and MR estimates attenuated slightly (Figures \@ref(fig:Obs-MR-without-top8) A/B). These results suggest causal effects of BMI across the general proteome.

```{r Estimate-comparison-obs, echo = FALSE, out.width = "90%", fig.cap="Scatter plot of the unadjusted (age and sex adjusted) observational estimates and the confounder-adjusted observational estimates for BMI and protein traits with a regression line.", fig.scap="Scatter plot comparing the unadjusted and confounder-adjusted observational BMI-protein estimates"}
include_graphics(path = "figure/BMI_protein_INTERVAL/obs_v_obs_adj_scatter.pdf", dpi = NA)

```

```{r Estimate-comparison-obs-mr, echo = FALSE, out.width = "90%", fig.cap="Scatter plot of the unadjusted (age and sex adjusted) observational estimates and the MR estimates for BMI and protein traits with a regression line", fig.scap = "Scatter plot comparing the observational and MR BMI-protein estimates"}
include_graphics(path = "figure/BMI_protein_INTERVAL/unadj_obs_MR_scatter_small.pdf", dpi = NA)

```

```{r Obs-MR-without-top8, echo = FALSE, out.width = "80%", fig.cap="A) Scatter plot of the unadjusted (age and sex adjusted) observational estimates and the confounder-adjusted observational estimates for BMI and protein traits with a regression line (blue), with the top eight MR BMI-associated proteins excluded. 6B) Scatter plot of the unadjusted (age and sex adjusted) observational estimates and the MR estimates for BMI and protein traits with a regression line (blue), with the top eight MR BMI-associated proteins excluded.", fig.scap="Scatter plot comparing BMI-protein estimates across models with top eight BMI-associated proteins excluded"}
include_graphics(path = "figure/BMI_protein_INTERVAL/Obs_MR_scatter_without_top8.pdf", dpi = NA)

```

### Enrichment analysis of strongest BMI-protein associations

In examining the clustering of proteins, visual representation using a scree plot suggested there were five PCs that explained 30.3% of the variance (Figure \@ref(fig:PCs)). After PC5 there was clear drop in variance explained, therefore other PCs were excluded. These five PCs were entered into a k-means analysis, which provided evidence for five clusters (grouping of individual proteins is included in Supplementary Table 7). To identify which cluster was most strongly affected by BMI, the median absolute beta coefficient divided by the SE for each cluster was compared with the overall estimate. Six of the proteins out of the eight strongest BMI-protein MR estimates were in cluster 2 (Supplementary Table 7). There was consistent evidence that cluster 2 showed a stronger association with BMI than the overall average BMI-protein effect both observationally (3.79 (IQR 1.62-7.06) vs 3.35 (IQR 1.57-5.83) respectively, P=3.7x10^-4^) and in MR (0.85 (IQR 0.41-1.46) vs 0.74 (IQR 0.32-1.14), P=5.3x10^-6^, Table \@ref(tab:Cluster-estimates)). Cluster 2 showed consistent evidence of a having the largest BMI effect. Compared with the full protein list in SomaLogic, the proteins in cluster 2 were enriched for disease (Table \@ref(tab:DAVID-enrichment-cluster2)), including cardiovascular disease (1.14 fold enrichment, P=1.3x10^-4^), renal disease (1.22 fold enrichment, P=1.0x10^-3^) cancer (1.1 fold enrichment, P=9.5x10^-3^) and metabolic disease (1.08 fold enrichment, P=4.2x10^-2^). No other individual cluster showed enrichment for disease. Enrichment for disease was also explored by comparing the proteins which had an association with BMI (P<1.4x10^-5^) in the confounder-adjusted regression model with the total protein list. Compared with the full protein list, the proteins which showed a stronger observational association with BMI were enriched for renal disease (1.21 fold enrichment, P=0.001) and metabolic disease (1.9 fold enrichment, P=0.015, Table \@ref(tab:DAVID-enrichment-obs)). 

```{r Cluster-estimates,  echo=FALSE}
options(knitr.kable.NA = "")
Cluster = c(1, 2, 3, 4, 5, "All")
N <- c(1061, 1406, 581, 502, 484, 4034)
Medobs <- c('3.28 (1.56-5.26)','3.79 (1.62-7.06)','2.77 (1.39-4.30)','3.01 (1.25-5.6)','4.44 (2.2-7.05)','3.35 (1.57-5.83)')
Pobs <- c(1, '3.70E-4', 1, 1, '1.80E-5', NA)
MedMR <- c('0.69 (0.31-1.22)','0.85 (0.41-1.46)','0.74 (0.37-1.25)','0.68 (0.34-1.19)','0.63 (0.32-1.14)','0.74 (0.35-1.28)')
PMR <- c(1, '5.3E-6', 1, 1, 1, NA)
mytable = data.frame(cbind(Cluster, N, Medobs, Pobs, MedMR, PMR))
colnames(mytable) <- c( 'Cluster',	'N',	'Median absolute observational beta coefficient divided by SE (IQR)',	'P value* for cluster v all proteins (Observational)',	'Median MR beta coefficient divided by SE (IQR)',	'P value* for cluster v all proteins (MR)')
mytable %>% kable(caption = "Comparison of cluster median(absolute beta coefficient / SE) with the overall median for observational and MR analyses, *One-tailed pairwise Wilcox test", booktabs = TRUE, caption.short = "Comparison of BMI-protein estimates among protein clusters") %>% 
  kable_classic_2(full_width = F) %>%
 kable_styling(full_width = TRUE) %>%
column_spec(3, width = "3cm")   %>%
 column_spec(5, width = "3cm")  %>%
  landscape()
```

```{r DAVID-enrichment-cluster2,  echo=FALSE}
options(knitr.kable.NA = "")
Term = c('Cardiovascular', 'Renal', 'Cancer', 'Pharmacogenomic', 'Metabolic', 'Vision', 'Haematological', 'Reproduction', 'Immune', 'Neurological', 'Aging')
Count = c(439,216,401,357,504,100,168,152,360,311,117)
Percentage = c(34.1, 16.8,31.2,27.8,39.2,7.8,13.1,11.8,28.0,24.2,9.1)
List <- c(1024, 1024, 1024, 1024, 1024, 1024, 1024, 1024, 1024, 1024, 1024 )
Pophits <- c(1023, 472,958,856,1243,216,388,351,886,759,278)
Poptotal <- c(2723, 2723, 2723, 2723, 2723, 2723 ,2723 ,2723 ,2723 ,2723 ,2723)
FE <- c(1.14, 1.22, 1.11, 1.11, 1.08, 1.23, 1.15, 1.15, 1.08, 1.09, 1.12)
Pval <- c('7.3E-06','5.8E-05','5.3E-04','1.9E-03','2.4E-03','5.9E-03','9.8E-03','1.4E-02','1.5E-02','1.6E-02','7.5E-02')
Bonf_adj_P <- c('1.3E-04','1.0E-03', '9.5E-03','3.4E-02','4.2E-02','1.0E-01','1.6E-01','2.3E-01','2.4E-01','2.5E-01','7.5E-01')
mytable = data.frame(cbind(Term, Count, Percentage, List, Pophits, Poptotal, FE, Pval, Bonf_adj_P))
colnames(mytable) <- c('Term (Genetic Association Database disease class)',	'Count',	'%',	'List Total','	Population Hits',	'Population Total',	'Fold Enrichment',	'P Value',	'Bonferroni-adjusted P value' )
mytable %>% kable(caption = "Cluster 2 vs full SomaLogic protein enrichment results for disease class using DAVID bioinformatics 6.8", booktabs = TRUE) %>% 
  kable_classic_2(full_width = F) %>%
 kable_styling(full_width = TRUE) %>%
  column_spec(1, width = "5cm")   %>%
  landscape()
```

```{r DAVID-enrichment-obs,  echo=FALSE}
options(knitr.kable.NA = "")
Term = c('Renal', 'Pharmacogenomic', 'Metabolic', 'Aging', 'Cardiovascular', 'Unknown', "Psych", 'Reproduction', "Normal variation", "Other", 'Cancer', 'Haematological', 'Neurological', "Developmental" )
Count = c(218,372,515,129,424,233,228,154,86,211,391,164,309,160)
Percentage = c(16.4,28.0,38.8,9.7,32.0,17.6,17.2,11.6,6.5,15.9,29.5,12.4,23.3,12.1)
List <- c(1039,1039,1039,1039,1039,1039,1039,1039,1039,1039,1039,1039,1039,1039 )
Pophits <- c(472,856,1243,278,1023,541,534,351,185,497,958,388,759,381)
Poptotal <- c(2723, 2723, 2723, 2723, 2723, 2723 ,2723 ,2723 ,2723 ,2723 ,2723, 2723, 2723, 2723)
FE <- c(1.21,1.14,1.09,1.22,1.09,1.13,1.12,1.15,1.22,1.11,1.07,1.11,1.07,1.10)
Pval <- c('7.7E-05','8.8E-05','8.3E-04','2.7E-03','4.1E-03','6.4E-03','1.2E-02','1.4E-02','1.5E-02','2.1E-02','2.2E-02','5.0E-02','5.5E-02','6.6E-02')
Bonf_adj_P <- c(0.001,0.002,0.015,0.047,0.072,0.109,0.189,0.228,0.236,0.313,0.332,0.605,0.638,0.708)
mytable = data.frame(cbind(Term, Count, Percentage, List, Pophits, Poptotal, FE, Pval, Bonf_adj_P))
colnames(mytable) <- c('Term (Genetic Association Database disease class)',	'Count',	'%',	'List Total','	Population Hits',	'Population Total',	'Fold Enrichment',	'P Value',	'Bonferroni-adjusted P value' )
mytable %>% kable(caption = "Disease enrichment of proteins associated with BMI in confounder adjusted model compared with all proteins in SomaLogic (DAVID Bioinformatics 6.8)", booktabs = TRUE) %>% 
  kable_classic_2(full_width = F) %>%
 kable_styling(full_width = TRUE) %>%
  column_spec(1, width = "5cm")   %>%
  landscape()
```

## Discussion

This chapter sought to estimate the effects of adiposity on a comprehensive set of protein traits only recently measurable by untargeted proteomics using observational and MR methods. This analysis was performed as alterations in the plasma proteins may have implications for cardiovascular disease, and therefore relating to platelet function. Observational results provided evidence for associations between BMI and 1576 proteins and MR was performed to reduce confounding. MR results suggest that BMI alters protein traits involved in regulating appetite, sex hormones, inflammation, and other systems; specific proteins most altered by BMI include leptin, FABP4, and SHBG. Results of follow-up analyses suggest that the cluster of proteins most altered by BMI is enriched for genes associated with cardiovascular and metabolic disease. 

This chapter explored the effect of BMI on a large set of circulating proteins in an MR framework. Previous studies have used observational epidemiology to explore the effect of obesity on the plasma proteome: one study used mass spectrometry and found an increase in Complement Factors I, B and H and an increase in CRP [@Cominetti2018]. These findings were replicated in our current observational analysis using the SomaLogic platform, indicating that associations are detectable across different proteomic platforms. The only association that did not replicate in the current analysis was the positive association with protein S100-A9. Although the MR analysis did not support some of these BMI-protein associations as being causal based on a P-value reference point, the strong association between the observational and MR estimates throughout the entire effect distribution suggests that disagreements between methods are likely an issue of power given current sample sizes.

Previous work implementing MR to examine the relationship between BMI and ~1000 proteins (measured using the same SomaLogic array) provided corroborative evidence to that shown here [@Zaghlool2021]. Both studies suggested a positive association between BMI and leptin, as well as a negative association with SHBG. Other proteins, such as IGFBP1/2 and growth hormone receptor, did not pass our multiple testing threshold, but the direction and magnitude of estimates were in agreement, suggesting a possible causal effect that was not detectable in the current analysis. Building on previous work, the current chapter provides MR estimates for >3600 proteins, offering a wider proteomic profile and detecting additional associations such as that between BMI and fumarylacetoacetase and inhibin beta C chain. Furthermore, the inclusion of over threefold more proteins allowed a more comprehensive enrichment analysis to be performed.

For proteins with stronger MR derived association evidence, it is important to explore whether they have a potential role in disease. Identification of individual proteins could help to guide future intervention if changes in proteins can be mapped to disease outcomes. Our results suggest a strong positive effect of BMI on levels of leptin, a hormone released by white adipose tissue which suppresses appetite [@Klok2007]. The direction of effect agrees with estimates from previous cross-sectional and MR studies [@Wurtz2014; @Millard2015], indicating leptin receptor resistance [@Gruzdeva2019a]. There is observational evidence in humans that higher leptin can induce greater aggregation of platelets (cells involved in haemostasis) [@Nakata1999]. In a larger observational study, leptin was found to be associated with higher risk of coronary events independent of BMI [@Wallace2001].

Our results help to provide contextualisation for proteins which have already been implicated in disease. For example, results suggest a strong positive effect of BMI on FABP4, an adipokine found primarily in adipocytes and macrophages [@Furuhashi2014]. This MR estimate supports the association which has been suggested in previous observational studies [@Xu2006]. FABP4 has been implicated in cardiometabolic disease: a SNP which increases FABP4 was found to raise the odds of type II diabetes among adults [@Gudmundsdottir2020], potentially through its contribution to higher insulin resistance [@Nakamura2017]. FABP4 has also been associated with higher risk of atherosclerosis among adults [@Yeung2007]. A strong SHBG-lowering effect of higher BMI was also suggested here. The SHBG molecule is a glycoprotein which binds androgens and oestrogens and suppresses their activity [@Wallace2013]; a reduction in SHBG is therefore expected to lead to higher levels of circulating sex hormones. The negative effect of BMI on SHBG seen here supports observational findings [@Cooper2015; @Goto2014; @Baglietto2009]. When evaluating the role of SHBG in disease, MR analysis suggests that an increase in SHBG contributes to a decrease in risk of cardioembolic stroke [@Zheng2020]. Other studies have also implicated lower SHBG levels in increasing type II diabetes risk  [@Ritchie2019]. The exact mechanisms leading from decreased SBHG to ill-health is unclear, but may arise as a result of the increased bioavailability of testosterone and oestrogen [@Wallace2013].

Despite these possible protein involvements in cardiometabolic disease, it remains difficult to assess the contribution of individual proteins as they are not entirely independent and any pathological effects would likely be due to a global change in protein composition. There are not distinct groupings in the SomaLogic data as there often are with, for example, metabolomics data. We therefore examined proteins grouped into clusters of similar features, compared BMI-protein estimates of each cluster with overall estimates and explored enrichment for genes related to disease. The cluster most altered by BMI (cluster 2) included most of the eight proteins with the strongest BMI effects from MR analyses, as well as various complement factors, chemokines and coagulation factors. Therefore, it is very likely that this global change in proteins would activate platelets. This cluster was also found to be enriched for genes related to cardiovascular disease, renal and metabolic diseases, and cancer. Enrichment was similar when comparing the proteins that had an observational association with BMI with all proteins included, with enrichment appearing greatest for renal and metabolic disease. Together, this suggests that changes in proteins may mediate effects of obesity on cardiometabolic diseases; more focused investigations of these proteins are now needed, especially to assess their impact on platelet function. When IPC GWAS summary stats become available, it would be useful to perform a PheWAS of protein exposures to IPC to explore whether any proteins which are altered by BMI can affect IPC.

This analysis has some limitations. Firstly, although INTERVAL is one of the largest existing cohorts to have untargeted proteomic data based on the SomaLogic platform, the sample size is still relatively modest and may have low power to detect some associations when using MR: based on the detectable (P<1.4x10~-5~) median absolute observational effect size (0.13 SDs), our analyses had 80% power to detect MR effect sizes >= 0.33 SDs (alpha=0.05) for our sample size (N=2737) [@Brion2013]. With greater statistical power, there would likely be more proteins detected with MR. This was reinforced by the strong agreement in the magnitude of effect estimates seen in observational and MR analyses which applied throughout the effect distribution. Secondly, height and weight were self-reported which could bias results towards the null due to systematic error in BMI measurement. However, strong correlations are often reported between self-reported and measured BMI [@Nikolaou2017] and the validity of self-reported BMI is supported by the association between the GRS for BMI and self-reported BMI in INTERVAL to the degree expected. Thirdly, the small degree of overlap between INTERVAL and UK Biobank (participants used for the source GWAS for BMI who were also in INTERVAL) may have a biasing effect on estimates, though this is likely to be towards the null. We anticipate that overall, this bias would make estimates more conservative [@Burgess2016]. Fourth, we recognise a lack of availability of possible confounders such as socio-economic position (which likely affect both BMI and protein traits related to cardiovascular disease processes [@Rosengren2019; @Tillmann2017]. Residual confounding may help account for the divergence between observed and expected P-values seen in observational versus MR models. Fifth, the proteins examined are highly correlated and we therefore may not fully be describing changes in individual proteins. Evidence from case-control cohorts as well as functional and animal studies would help isolate individual proteins that are altered and contribute to disease. Finally, although analyses provide insight into the proteomic effects of BMI, it does not distinguish between the type of adiposity. It would be useful to distinguish between the effects of subcutaneous and visceral fat using dual-energy X-ray absorptiometry (DXA) derived measurements, but these were not available in the INTERVAL dataset.

This chapter utilised SomaLogic to explore the relationship between BMI and plasma proteins in unprecendented scope and detail, in both an observational and MR framework. There is evidence for a broad impact of higher adiposity on the human proteome. Causal evidence was strongest for BMI in relation to proteins involved in regulating appetite, sex hormones, and inflammation. Protein alterations were found to be enriched for genes related to cardiovascular and metabolic disease. Altogether, these results help to focus attention onto new potential proteomic signatures of obesity-related disease. Further characterisation of the role of such proteomic profiles in cardiovascular disease using MR is warranted, as well as characterising the role of such proteins on platelet function.
